src/main/java/net/earthcomputer/clientcommands/command/GhostBlockCommand.java

```diff
@@ -49,7 +57,7 @@ private static int setGhostBlock(FabricClientCommandSource source, BlockPos pos,
         }
     }
 
-    private static int fillGhostBlocks(FabricClientCommandSource source, BlockPos from, BlockPos to, BlockState state) throws CommandSyntaxException {
+    private static int fillGhostBlocks(FabricClientCommandSource source, BlockPos from, BlockPos to, BlockState state, @Nullable Predicate<CachedBlockPosition> filter) throws CommandSyntaxException {
```

Comment:
Any reason this predicate has to be nullable instead of using `pos -> true` instead of `null`?